### ISSUE TITLE
Causes invalid peer dependencies

### ISSUE THREAD

**Issue by abrkn** [peerigon/updtr#11]
```
$ updtr
Found 9 outdated modules

Starting to update your modules...
1/9 	 ● bluebird 3.0.5 success
2/9 	 ● bower 1.6.8 success
3/9 	 ● imports-loader 0.6.5 success
4/9 	 ● koa 1.1.2 success
5/9 	 ● koa-etag 2.1.0 success
6/9 	 ● koa-static 1.5.2 success
7/9 	 ● qs 6.0.1 success
8/9 	 ● socket.io-client 1.3.7 success
9/9 	 ● stylus-loader 1.4.2 success

Finished
```

```
$ npm shrinkwrap
npm WARN unmet dependency --snip--/node_modules/history requires qs@'^4.0.0' but will load
npm WARN unmet dependency --snip--/node_modules/qs,
npm WARN unmet dependency which is version 6.0.1
npm WARN unmet dependency --snip--/node_modules/babel-core requires bluebird@'^2.9.33' but will load
npm WARN unmet dependency --snip--/node_modules/bluebird,
npm WARN unmet dependency which is version 3.0.5
npm WARN unmet dependency --snip--/node_modules/redux-router/node_modules/history requires qs@'^4.0.0' but will load
npm WARN unmet dependency --snip--/node_modules/qs,
npm WARN unmet dependency which is version 6.0.1
npm WARN unmet dependency --snip--/node_modules/webpack-dev-server/node_modules/express requires qs@'4.0.0' but will load
npm WARN unmet dependency --snip--/node_modules/qs,
npm WARN unmet dependency which is version 6.0.1
npm ERR! Darwin 14.5.0
npm ERR! argv "/Users/andreas/.nvm/versions/io.js/v2.3.4/bin/iojs" "/Users/andreas/.nvm/versions/io.js/v2.3.4/bin/npm" "shrinkwrap"
npm ERR! node v2.3.4
npm ERR! npm  v2.12.1

npm ERR! Problems were encountered
npm ERR! Please correct and try again.
npm ERR! invalid: bluebird@3.0.5 --snip--/node_modules/bluebird
npm ERR! invalid: qs@6.0.1 --snip--/node_modules/qs
npm ERR!
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     --snip--/npm-debug.log
```

**Comment by abrkn** [comment]
How about a command-line switch that lets the user specify which command updtr should run to test? In my case, that would be: `npm shrinkwrap & webpack ... & npm test`

**Comment by BloodyHell** [comment]
I've found that by default it runs npm test.
You can specify which test code to run by defining the following code in your package.json:

```javascript
"scripts": {
    "test": npm shrinkwrap & webpack ... & npm test 
}
```

**Comment by abrkn** [comment]
:+1:

**Comment by matthaias** [comment]
> :+1: 

Does that mean, that your problem is solved with @BloodyHell's answer?

**Comment by abrkn** [comment]
It would still be better if it preferred to run a script named `updtr` with fallback to `test` script.

**Comment by matthaias** [comment]
Ok, or maybe that the user can choose the command, for example: `$ updtr --test updtr`?

**Comment by matthaias** [comment]
Please check out branch `feature/custom-test-command` and test if it what you need.

Example:
```
$ updtr --test "npm run customtestcommand"
```

**Comment by matthaias** [comment]
Published as `v0.1.6`
